elastic/beats
# [Azure Billing] Update forecast api metrics interval

- Bug


## What does this PR do?

Fixes #35633 

## Why is it important?

The proposed change fetches the forecast data for the next 30 days from the current day(-2). As of today, the forecast metrics is fetched from the beginning of the month to the end of the month. Considering the fact that this API returns the actual data to consolidate the cost we are adjusting the start date to current day - 2 days.

## How to test this PR locally

- Enable the Azure billing metrics module.
- Perform the configuration 
- Verify the billing metrics forecast API fetches metrics from yesterday and for next 30 days.
- Ensure there the other metrics are getting loaded in the dashboards.

## Related issues

- Closes #35633

## Screenshots
<img width="1695" alt="Screenshot 2023-07-24 at 1 00 30 PM" src="https://github.com/elastic/beats/assets/101238137/c0d3544f-d3a4-422b-9fff-185945b46512">
